app/main.py

```diff
@@ -1 +1,34 @@
-#  write your code here
+class Animal():
+    def __init__(self, name: str, appetite: int, is_hungry: bool = True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+class Cat(Animal):
+    def __init__(self, name: str, appetite=3, is_hungry: bool = True):
```

Comment:
The `Cat` class should not accept an `appetite` parameter. It should automatically set `appetite` to 3 when calling the superclass `__init__` method.